bindings/nodejs/lib/types/block/context_input/context_input.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { AccountId } from '../..';
+import { u16 } from '../../utils/type_aliases';
+
+enum ContextInputType {
```

Comment:
Shouldn't be called `ContextInputKind`. I mean in rust the word `kind` is used https://github.com/abdulmth/iota-sdk/blob/ccde4dbf229ed7e66f471cf73308d975442225bc/sdk/src/types/block/context_input/block_issuance_credit.rs#L14

Reply:
I think in the node bindings the word `type` is used ([example `InputType`](https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/lib/types/block/input/input.ts#L10)), and the `kind` word from Rust is serialized to `type`. Maybe `kind` is used in Rust because `type` is a reserved keyword?